include/seqan3/argument_parser/detail/format_parse.hpp

```diff
@@ -202,16 +202,25 @@ class format_parse : public format_base
     }
 
     /*!\brief Finds the position of a short/long identifier in format_parse::argv.
-     *
+     * \tparam id_type The identifier type. If it is `char`, the parameter `id` is assumed to be a short identifier,
+                       on the other hand, if it is `std::string`, the parameter `id` is assumed to be a long identifier.
      * \param[in] begin_it The iterator where to start the search of the identifier.
      *                     Note that the end iterator is kept as a member variable.
      * \param[in] id       The identifier to search for (must not contain dashes).
      * \returns An iterator pointing to the first occurrence of the identifier in
      *          the list pointed to by begin_t. If the list does not contain the
      *          identifier `id`, the member variable `end_of_options_it` is returned.
      *
-     * Note: The `id` is compared to the prefix of each value in the list, such
-     *       that "-idValue" arguments are correctly identified.
+     * \details
+     *
+     * **Valid short-id value pairs are: `-iValue`, `-i=Value`, or `-i Value`**
+     * If the `id` passed to this function is of type `char` it is assumed to be a short identifier.
```

Comment:
```suggestion
     * If the `id` passed to this function is of type `char`, it is assumed to be a short identifier.
```

Reply:
You can also do multi-line suggestions :)